### ISSUE TITLE
[BUG]: Hero image needs descriptive alternetive text.

### ISSUE THREAD

**Issue by colabottles** [TechIsHiring/techishiring-website#154]
Home page hero image needs updated and descriptive alt text to convey the image properly to screen reader users.

## Expected Behavior

Alternative text for the hero image should be descriptive and convey what the image context contains.

## Context (Environment)

Screen reader users will benefit from more descriptive alt text in the image when the image is conveyed back through to the user by the screen reader.

## Possible Implementation

`alt="A young, Black man taking notes in front of a computer."` (spelling, grammar, and punctuation are important in alternative text for accessibility reasons).

**Comment by Necolanch** [comment]
@chadstewart I see you are assigned to this. I could quickly update this alt text if you'd like.
